MLS.Agent.Tests/Markdown/CodeBlockAnnotationExtensionTests.cs

```diff
@@ -90,27 +90,34 @@ public async Task Does_not_insert_code_when_specified_language_is_not_csharp()
             html.Should().Contain(expectedValue);
         }
 
-        [Fact]
-        public async Task Does_not_insert_code_when_csharp_is_specified_but_no_additional_options()
+        [Theory]
+        [InlineData("cs", "language-cs")]
+        [InlineData("csharp", "language-csharp")]
+        [InlineData("c#", "language-c#")]
+        [InlineData("fs", "language-fs")]
+        [InlineData("fsharp", "language-fsharp")]
+        [InlineData("f#", "language-f#")]
+        public async Task Does_not_insert_code_when_supported_language_is_specified_but_no_additional_options(string fenceLanguage, string expectedClass)
         {
-            string expectedValue =
-@"<pre><code class=""language-cs"">Console.WriteLine(&quot;Hello World&quot;);
+            var expectedValue =
+$@"<pre><code class=""{expectedClass}"">Console.WriteLine(&quot;Hello World&quot;);
```

Comment:
Should we not normalize these so that the output class is always the same for the various   inputs from the fence?

Reply:
sounds a good idea.